RiotSwiftUI/Modules/Room/PollHistory/Service/PollHistoryServiceProtocol.swift

```diff
@@ -14,6 +14,17 @@
 // limitations under the License.
 //
 
+import Combine
+
 protocol PollHistoryServiceProtocol {
-    func fetchHistory() async throws -> [PollListData]
+    /// Returns a Publisher publishing the polls in the next batch.
+    /// Implementations should return the same publisher if `next()` is called again before the previous publisher completes.
+    func next() -> AnyPublisher<TimelinePollDetails, Error>
+    
+    /// Publishes updates for the polls previously pusblished by the `next()` publishers.
+    var updates: AnyPublisher<TimelinePollDetails, Never> { get }
+    
+    /// Publishes errors regarding poll aggregations.
+    /// Note: `next()` will continue to publish new polls even if some poll isn't being aggregated correctly.
+    var pollErrors: AnyPublisher<Error, Never> { get }
```

Comment:
I don't think this practice adds value to our codebase. On the contrary increase verbosity. 
It would be like suffixing "sequence" every time you have an array (or another Sequence) property.
Also Apple seems not to do that [here](https://developer.apple.com/documentation/Combine/ObservableObject). 🤔

Reply:
Yep yep, seeing it in this PR I do agree, the call site looks way nicer without the suffix.